yarn.lock

```diff
@@ -1873,10 +1873,10 @@
     schema-utils "^3.0.0"
```

Comment:
As @Carnageous correctly pointed out, I expect the yarn.lock to not change in this PR.